modules/core/shared/src/main/scala/lucuma/core/conditions/package.scala

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.conditions
+
+import coulomb.Quantity
+import coulomb.units.accepted.*
+import eu.timepit.refined.*
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.boolean.Not
+import eu.timepit.refined.numeric.Interval as RefinedInterval
+import eu.timepit.refined.numeric.Less
+import lucuma.core.enums.CloudExtinction
+import lucuma.core.enums.Site
+import lucuma.core.enums.SkyBackground
+import lucuma.core.enums.WaterVapor
+import lucuma.core.math.Declination
+import lucuma.core.math.Wavelength
+import lucuma.core.math.erf
+import lucuma.core.util.RefinedNewType
+import lucuma.refined.*
+
+import scala.math.Pi
+import scala.math.abs
+import scala.math.pow
+import scala.math.sin
+
+// Calculations of likelihood of occurrence of observing conditions
+// Taken from:
+// https://github.com/andrewwstephens/pyexplore/blob/3edd50f6c41509752cda6ad493ccaadd5eb5ad82/test/percentile.py
+//
+// Store a percentage with two decimals of precision for a 0-100% range
+type CentiPercent    = RefinedInterval.Closed[0, 10000]
+object IntCentiPercent extends RefinedNewType[Int, CentiPercent]:
```

Comment:
Seems like it could go with `IntPercent` in `lucuma.core.model`?

Reply:
Right I could move it there